rdt/transformers/categorical.py

```diff
@@ -376,18 +369,19 @@ def _transform(self, data):
                 Data to transform.
 
         Returns:
-            numpy.ndarray:
+            numpy.ndarray.
         """
         data = self._prepare_data(data)
-        array = self._transform_helper(data)
-
-        if self.error_on_unknown:
-            unknown = array.sum(axis=1) == 0
-            if unknown.any():
-                raise ValueError(f'Attempted to transform {list(data[unknown])} ',
-                                 'that were not seen during fit stage.')
+        unique_data = {np.nan if pd.isna(x) else x for x in pd.unique(data)}
```

Comment:
very minor but we could make a module level function for this since it seems like it's used in each categorical transformer. It's a small thing though so it doesn't matter too much

Reply:
Yep, implemented in the `LabelEncoder` PR. Will update here when that's merged.